src/index.js

```diff
@@ -1,14 +1,32 @@
+/* eslint-disable no-unused-vars */
 import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const tasks = [{ description: 'wash the dishes', completed: false, index: 0 },
+  { description: 'complete To Do list project', completed: false, index: 1 }];
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+const target = document.getElementById('listContainer');
+target.innerHTML = `
+<li class ="commonClass">
+<label class ="checkBoxLabel">
+<input for ="job" id="inputField" type="checkbox" class ="checkBox">
+<p id ="job" class ="pInsideLi">${tasks[0].description}</p>
+</label>
+<button  class="btnAll threeDots">
+ <i class="fa fa-ellipsis-h"></i>
+</button>
+</li>
+<li class ="commonClass">
+<label class ="checkBoxLabel">
+<input for ="job2" id="inputField" type="checkbox" class ="checkBox">
+<p id ="job2" class ="pInsideLi">${tasks[1].description}</p>
+</label>
+<button  class="btnAll threeDots">
+ <i class="fa fa-ellipsis-h"></i>
+</button>
+</li>
```

Comment:
- [ ] Kindly manipulate the DOM dynamically as raised earlier, you only need the first portion of the code written above to achieve this. More so, there is a need to loop through the task array in order to access the content of the array. 
```suggestion
const target = document.getElementById('listContainer');
for (let i = 0; i < tasks.length; i += 1) {
  target.innerHTML += `
<li class ="commonClass">
<label class ="checkBoxLabel">
<input for ="job" id="inputField" type="checkbox" class ="checkBox">
<p id ="job" class ="pInsideLi">${tasks[i].description}</p>
</label>
<button  class="btnAll threeDots">
 <i class="fa fa-ellipsis-h"></i>
</button>
</li>
`;
}
```

- [ ] Kindly ensure that the list appear in order of the index values for each task.

Reply:
resolved